ShopAPI/ShopAPI/Services/ProductService.cs

```diff
@@ -119,11 +119,15 @@ public async Task<Product> UpdateProductPriceAsync(int id, decimal price)
         /// <returns>Nothing.</returns>
         public async Task<Product> AddProductStockAsync(int id, int quantity)
         {
+            if (quantity < 0 || quantity == null)
+            {
+                throw new ConflictException("Quantity must be greater than or equal to 0");
```

Comment:
Should be [422 Unprocessable Content](https://www.rfc-editor.org/rfc/rfc9110.html#name-422-unprocessable-content)

Reply:
Done